src/Tests/VCEL.Test/ToJSCodeTests.cs

```diff
@@ -27,6 +27,8 @@ public ToJsCodeTests()
     [TestCase("(D > 500000 or D < -500000)", "((vcelContext.D > 500000) || (vcelContext.D < -500000))")]
     [TestCase("(D > 123456789)", "(vcelContext.D > 123456789)")]
     [TestCase("(D > 1234567890)", "(vcelContext.D > 1234567890)")]
+    [TestCase("(_.env.user == 'jdoe2')", "(_.env.user === 'jdoe2')")]
+    [TestCase("(env._.user == 'jdoe2')", "(vcelContext.env._.user === 'jdoe2')")]
```

Comment:
can we add more test case to other categories.
e.g. in expr, match expr, let expr etc...

Reply:
Sure, added!